src/index.js

```diff
@@ -18,40 +19,106 @@ function runPackageScripts({scriptConfig, scripts, args, options = {}}, callback
     scripts = ['default']
   }
   const scriptNames = arrify(scripts)
-  const method = options.parallel ? 'map' : 'mapSeries'
-  async[method](scriptNames, (scriptName, cb) => {
-    const child = runPackageScript({scriptConfig, options, scriptName, args})
-    if (child.on) {
-      child.on('exit', exitCode => cb(null, exitCode))
-    } else {
-      cb(child)
+  if (options.parallel) {
```

Comment:
I don't know what I'm looking for.. I just think this code could be cleaned up a bit...